lib/copy-sync/copy-sync.js

```diff
@@ -91,7 +98,10 @@ function copyFileFallback (srcStat, src, dest, opts) {
     pos += bytesRead
   }
 
-  if (opts.preserveTimestamps) fs.futimesSync(fdw, srcStat.atime, srcStat.mtime)
+  if (opts.preserveTimestamps) {
+    const updatedSrcStat = fs.statSync(src)
+    fs.futimesSync(dest, updatedSrcStat.atime, updatedSrcStat.mtime)
```

Comment:
`fs.futimesSync()` should use `fdw` and not `dest`.

Reply:
Oops, absolutely!

Reply:
Seems like no test verified this code path. Let me check.

Reply:
Indeed - `copyFileFallback` in both `copy.js` and `copySync.js` is not covered at all, according to coveralls.

Reply:
Fixed in bd947fdc12c1be6c65206bac45794bbe48d6041f